content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,90 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
```

Comment:
**Added** experimental `manifest` and `profile` commands for evaluation purpose

Reply:
@linsun do you mean to replace line 3 or an addendum?

Reply:
replace line 3 s/a command line/experimental manifest and profile commands for evaluation purpose/

Reply:
@linsun if you are satisfied with the latest text, please click the `Resolve conversation` button.